docs/source/software/spack.md

```diff
@@ -45,6 +55,8 @@ This section details how to install the LAMMPS application with the KOKKOS and U
    srun --partition=gpu --nodes=1 --ntasks=14 --pty --export=All --gres=gpu:1 --mem=0 --time=08:00:00 /bin/bash
```

Comment:
I am not sure if --mem=0 is best practice here because you are pulling all memory on the node and leaving GPUs on the node that will sit idle.